postgresql.sql

```diff
@@ -0,0 +1,15 @@
+--http://sqlfiddle.com/#!15/9eecb7db59d16c80417c72d1e1f4fbf1/11608
+--Runs with postgresql 9.3
```

Comment:
Honestly, I put 9.3 because I don't have postgresql readily available and I ran it on SQLFiddle which had only 9.3. But I'm sure there is nothing that will break in any 9.3+ version, its a simple piece of code